proto/ibc/apps/cross-chain-validation/tx.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package ibc.apps.ccv.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/ccv/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// MsgSubmitNewChain defines a msg to submit a new baby chain that can be validated
+// by the parent chain validators.
+message MsgSubmitNewChain {
+  option (gogoproto.goproto_getters) = false;
+  
+  // the proposed chain-id of new chain
+  string chain_id = 1 [(gogoproto.moretags) = "yaml:\"chain_id\""];
+  // minimum stake that must commit to validate new chain before it can start
+  cosmos.base.v1beta1.Coin minimum_stake = 2 [(gogoproto.nullable) = false];
+  // optional grace period to allow validators to join initial validator set after minimum stake is reached
+  // grace period is a duration in nanoseconds
+  uint64 grace_period = 3 [(gogoproto.moretags) = "yaml:\"grace_period\""];
+  // hash of the genesis file with no staking genesis. Staking genesis will be filled in using gen-txs of initial validator set
+  bytes genesis_hash = 4 [(gogoproto.moretags) = "yaml:\"genesis_hash\""];
+  // the sender address
+  string sender = 5;
+}
+
+// MsgSubmitNewChainResponse defines the Msg/SubmitNewChain response type.
+message MsgSubmitNewChainResponse {}
+
+// MsgBondStake defines a msg for a parent chain validator to stake on a baby chain.
+message MsgBondStake {
```

Comment:
These are the bonding/unbonding messages currently operating at the validator level. Though future improvements to staking module may push this to delegator level.

Reply:
I'd keep the (un)delegation instead of (un)bond for consistency with the staking module

Reply:
I've kept it bond/unbond because this was an action taken at the validator level and not at delegator level, so I didn't want to confuse the two. I can change if there's consensus to keep it the same.

Reply:
+1 to keeping with bond/unbond terminology